atlp-rwanda/eagles-phantom-backend
# Postgres databse configuration configuration

What does this PR does?
-sequelizing postgre database for testing configuration with Users table.

Description of task to be completed
-install sequelizing package and initialize it
-create database using mysql and get connection string
-create table Users for database
-migrate the the database 
-seed the table
-select all from table to see if seeded

How should be manually tested?

-create test database and Users tables with name, email, phone, createAt and updateAt
-install package `npm install sequelize sequelize-cli pg pg-hstore`
-add users through seeding  by editing folder Users file 
-then create user` sequelize db:seed:all`
-final see if have been added into tables 

Revant trello Story

https://trello.com/c/3RYihb3M/17-configure-the-app-to-use-postgres-database

![database](https://user-images.githubusercontent.com/42033331/94019594-401efc80-fdb2-11ea-931c-12c90b5e5db8.jpg)
